src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -330,5 +330,6 @@ public net.minecraft.world.level.storage.loot.functions.SetWritableBookPagesFunc
 public net.minecraft.world.level.storage.loot.functions.SetWrittenBookPagesFunction <init>(Ljava/util/List;Ljava/util/List;Lnet/minecraft/world/level/storage/loot/functions/ListOperation;)V # <init>
 public net.minecraft.world.level.storage.loot.functions.ToggleTooltips <init>(Ljava/util/List;Ljava/util/Map;)V # <init>
 
+public net.minecraft.world.entity.ai.sensing.VillagerHostilesSensor ACCEPTABLE_DISTANCE_FROM_HOSTILES
```

Comment:
Prefer using reflection in the datamaps provider instead of an AT if the field isn't already public in vanilla.